debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCClassArrayClassSlotIterator.java

```diff
@@ -47,7 +48,7 @@ protected GCClassArrayClassSlotIterator(J9ClassPointer clazz) throws CorruptData
 			slots.add(slot);
 			addresses.add(VoidPointer.cast(clazz.arrayClassEA()));
 		}
-		if(clazz.romClass().modifiers().allBitsIn(J9_JAVA_CLASS_ARRAY)) {
+		if(clazz.romClass().modifiers().allBitsIn(J9AccClassArray)) {
```

Comment:
This will break the ability to look at core files generated before this change.

Reply:
Should I be reverting the macro name change here? Or is there a better fix to this problem?

Reply:
I spoke with @tajila who assures me that the values in the two classes are equal and so this should be compatible with older core files; I'll resume my review...